metaci/release/utils.py

```diff
@@ -124,11 +131,48 @@ def send_release_webhook(project_config, release, config_item=None):
         raise Exception("\n".join(err["message"] for err in result["errors"]))
 
 
-def send_start_webhook(project_config, release, role, config_item):
+def send_submit_webhook(release, config_item=None):
+    if (
+        release is None
+        or not settings.METACI_RELEASE_WEBHOOK_URL
+        or not settings.METACI_START_STOP_WEBHOOK
+        or not settings.GUS_BUS_OWNER_ID
+        or not config_item
+    ):
+        return
+    logger.info(
+        f"Sending submit webhook for {release} to {settings.METACI_RELEASE_WEBHOOK_URL}"
+    )
+
+    payload = {"case_id": release.change_case_link}
+    token = jwt_for_webhook()
+    response = requests.post(
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/case/{release.change_case_link}/submit",
+        json=payload,
+        headers={"Authorization": f"Bearer {token}"},
+    )
+    result = response.json()
+    if (
+        "results" in result
+        and len(result["results"]) > 0
+        and result["results"][0]["success"]
+        or result["success"]
+    ):
+        return
+    else:
+        if "results" in result:
+            for error in result["results"]:
+                raise Exception("\n".join(err["message"] for err in error["errors"]))
+        else:
+            raise Exception("\n".join(err["message"] for err in result["errors"]))
+
+
+def send_start_webhook(release, role, config_item):
     if (
         release is None
         or not settings.METACI_RELEASE_WEBHOOK_URL
         or not settings.METACI_START_STOP_WEBHOOK
+        or not settings.GUS_BUS_OWNER_ID
```

Comment:
Don't need this check; the owner was set when the implementation step was created.